### ISSUE TITLE
Busch-Jaeger 6735/6736/6737: Zigbee control of relais/dimmer very unstable after polling interval has been made configurable

### ISSUE THREAD

**Issue by Ra72xx** [Koenkk/zigbee-herdsman-converters#7939]
When trying to operate the internal switch/relais of these Busch-Jaeger devices by Zigbee, it is almost unusable, as there are long delays, timeouts and errors. Not even half of the Zigbee commands are carried out as expected.  

This only affects the internal relais/dimmer, direct operation with the switch panel and also Zigbee usage of the additional switch rows is working normally.

This behaviour is observable since pull request https://github.com/Koenkk/zigbee-herdsman-converters/pull/7747

Current setup: 
- Zigbee2MQTT-Version 1.40.0 commit: 6fa1dab
- Coordinator-Typ EmberZNet
- Coordinator-Version 7.4.2 [GA]
- Frontend-Version 0.7.4
- Zigbee Herdsman Konverter Version 20.8.4
- Zigbee Herdsman Version 0.57.1
- Running in a podman container on x86-64 Ubuntu
- Zigbee dongle on USB 2.0 port with long cable

This bug report is a follow-up of https://github.com/Koenkk/zigbee-herdsman-converters/issues/7733, as the issue only occured after this issue had been fixed. I also opened a seperate issue on the regular Z2M issue tracker,  https://github.com/Koenkk/zigbee2mqtt/issues/23347, which got no result at all.

In the following log, all entries named "...Dimmer-Bedienelement..." and "...Relais-Bedienelement..." are such Busch-Jaeger devices. "WZ: Relais-Bedienelement Kombi Tür links" is operated and reacts with several seconds delay, while "WZ: Dimmer-Bedienelement Kombi Tür links" operated a few seconds later does not react at all. The log apparently contains errors in both cases.

All other devices in the log are not being actively operated. I'm also a bit surprised that the log contains so many entries for those devices nevertheless, as I have increased the polling intervall already.

Another observation: All those Busch-Jaeger devices are controlled as if they are dimmers, i.e. the brightness is handled as controllable, however many of my devices are only simple on/off switches. I don't know if this is related.
 
```

024-09-04 05:21:48z2m: Successfully changed options
info 2024-09-04 05:21:48z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/bridge/response/options', payload '{"data":{"restart_required":false},"status":"ok","transaction":"oiaif-1"}'
info 2024-09-04 05:21:48z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/FZ: Dimmer-Bedienelement-Kombi Tür', payload '{"brightness":254,"brightness_relay":54,"device":{"applicationVersion":1,"dateCode":"20161209","friendlyName":"FZ: Dimmer-Bedienelement-Kombi Tür","hardwareVersion":0,"ieeeAddr":"0xd85def11a1006a50","manufacturerID":4398,"manufacturerName":"Busch-Jaeger","model":"6735/6736/6737","networkAddress":37580,"powerSource":"Mains (single phase)","softwareBuildID":"1.2.0","stackVersion":1,"type":"Router","zclVersion":1},"last_seen":"2024-09-04T03:21:48.358Z","linkquality":192,"state":"ON","state_relay":"OFF"}'
info 2024-09-04 05:21:48z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/FZ: Dimmer-Bedienelement-Kombi Tür/relay', payload '{"brightness":54,"state":"OFF"}'
info 2024-09-04 05:21:49z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/FZ: Dimmer-Bedienelement-Kombi Tür', payload '{"brightness":254,"brightness_relay":54,"device":{"applicationVersion":1,"dateCode":"20161209","friendlyName":"FZ: Dimmer-Bedienelement-Kombi Tür","hardwareVersion":0,"ieeeAddr":"0xd85def11a1006a50","manufacturerID":4398,"manufacturerName":"Busch-Jaeger","model":"6735/6736/6737","networkAddress":37580,"powerSource":"Mains (single phase)","softwareBuildID":"1.2.0","stackVersion":1,"type":"Router","zclVersion":1},"last_seen":"2024-09-04T03:21:48.919Z","linkquality":188,"state":"ON","state_relay":"OFF"}'
info 2024-09-04 05:21:49z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/FZ: Dimmer-Bedienelement-Kombi Tür/relay', payload '{"brightness":54,"state":"OFF"}'
info 2024-09-04 05:21:49z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/K: Relais-Bedienelement-Kombi Tür', payload '{"brightness_relay":254,"device":{"applicationVersion":1,"dateCode":"20161209","friendlyName":"K: Relais-Bedienelement-Kombi Tür","hardwareVersion":0,"ieeeAddr":"0xd85def11a100466a","manufacturerID":4398,"manufacturerName":"Busch-Jaeger","model":"6735/6736/6737","networkAddress":24447,"powerSource":"Mains (single phase)","softwareBuildID":"1.2.0","stackVersion":1,"type":"Router","zclVersion":1},"last_seen":"2024-09-04T03:21:49.011Z","linkquality":168,"state":"OFF","state_relay":"OFF"}'
info 2024-09-04 05:21:49z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/K: Relais-Bedienelement-Kombi Tür/relay', payload '{"brightness":254,"state":"OFF"}'
info 2024-09-04 05:21:49z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/K: Relais-Bedienelement-Kombi Tür', payload '{"brightness_relay":254,"device":{"applicationVersion":1,"dateCode":"20161209","friendlyName":"K: Relais-Bedienelement-Kombi Tür","hardwareVersion":0,"ieeeAddr":"0xd85def11a100466a","manufacturerID":4398,"manufacturerName":"Busch-Jaeger","model":"6735/6736/6737","networkAddress":24447,"powerSource":"Mains (single phase)","softwareBuildID":"1.2.0","stackVersion":1,"type":"Router","zclVersion":1},"last_seen":"2024-09-04T03:21:49.092Z","linkquality":168,"state":"OFF","state_relay":"OFF"}'
info 2024-09-04 05:21:49z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/K: Relais-Bedienelement-Kombi Tür/relay', payload '{"brightness":254,"state":"OFF"}'
info 2024-09-04 05:21:50z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/FZ: Dimmer-Bedienelement-Kombi Tür', payload '{"brightness":254,"brightness_relay":54,"device":{"applicationVersion":1,"dateCode":"20161209","friendlyName":"FZ: Dimmer-Bedienelement-Kombi Tür","hardwareVersion":0,"ieeeAddr":"0xd85def11a1006a50","manufacturerID":4398,"manufacturerName":"Busch-Jaeger","model":"6735/6736/6737","networkAddress":37580,"powerSource":"Mains (single phase)","softwareBuildID":"1.2.0","stackVersion":1,"type":"Router","zclVersion":1},"last_seen":"2024-09-04T03:21:50.015Z","linkquality":192,"state":"ON","state_relay":"OFF"}'
info 2024-09-04 05:21:50z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/FZ: Dimmer-Bedienelement-Kombi Tür/relay', payload '{"brightness":54,"state":"OFF"}'
info 2024-09-04 05:21:50z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/FZ: Dimmer-Bedienelement-Kombi Tür', payload '{"brightness":254,"brightness_relay":54,"device":{"applicationVersion":1,"dateCode":"20161209","friendlyName":"FZ: Dimmer-Bedienelement-Kombi Tür","hardwareVersion":0,"ieeeAddr":"0xd85def11a1006a50","manufacturerID":4398,"manufacturerName":"Busch-Jaeger","model":"6735/6736/6737","networkAddress":37580,"powerSource":"Mains (single phase)","softwareBuildID":"1.2.0","stackVersion":1,"type":"Router","zclVersion":1},"last_seen":"2024-09-04T03:21:50.729Z","linkquality":192,"state":"ON","state_relay":"OFF"}'
info 2024-09-04 05:21:50z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/FZ: Dimmer-Bedienelement-Kombi Tür/relay', payload '{"brightness":54,"state":"OFF"}'
info 2024-09-04 05:21:57z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/K: Relais-Bedienelement-Kombi rechts', payload '{"brightness_relay":254,"device":{"applicationVersion":1,"dateCode":"20161209","friendlyName":"K: Relais-Bedienelement-Kombi rechts","hardwareVersion":0,"ieeeAddr":"0xd85def11a100469e","manufacturerID":4398,"manufacturerName":"Busch-Jaeger","model":"6735/6736/6737","networkAddress":65377,"powerSource":"Mains (single phase)","softwareBuildID":"1.2.0","stackVersion":1,"type":"Router","zclVersion":1},"last_seen":"2024-09-04T03:21:57.563Z","linkquality":172,"state":"OFF","state_relay":"OFF"}'
info 2024-09-04 05:21:57z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/K: Relais-Bedienelement-Kombi rechts/relay', payload '{"brightness":254,"state":"OFF"}'
info 2024-09-04 05:21:57z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/B: Thermostat', payload '{"away_preset_temperature":"5.0","battery":100,"calibrate":null,"calibrated":true,"child_lock":true,"device":{"applicationVersion":30,"dateCode":"Feb 20 2023","friendlyName":"B: Thermostat","hardwareVersion":1,"ieeeAddr":"0x54ef4410007043f7","manufacturerID":4447,"manufacturerName":"LUMI","model":"SRTS-A01","networkAddress":16885,"powerSource":"Battery","softwareBuildID":"0.0.0_1030","type":"EndDevice","zclVersion":3},"device_temperature":27,"external_temperature_input":null,"internal_heating_setpoint":5,"last_seen":"2024-09-04T03:21:57.797Z","linkquality":255,"local_temperature":22.9,"occupied_heating_setpoint":22.5,"power_outage_count":4,"preset":"manual","schedule":false,"schedule_settings":"mon,sun|0:00,21.0|12:00,17.0|18:00,21.0|23:59,21.0","sensor":"external","setup":false,"system_mode":"off","update":{"installed_version":2590,"latest_version":2590,"state":"idle"},"update_available":null,"valve_alarm":true,"valve_detection":true,"voltage":2700,"window_detection":true,"window_open":false}'
info 2024-09-04 05:21:57z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/K: Relais-Bedienelement-Kombi rechts', payload '{"brightness_relay":254,"device":{"applicationVersion":1,"dateCode":"20161209","friendlyName":"K: Relais-Bedienelement-Kombi rechts","hardwareVersion":0,"ieeeAddr":"0xd85def11a100469e","manufacturerID":4398,"manufacturerName":"Busch-Jaeger","model":"6735/6736/6737","networkAddress":65377,"powerSource":"Mains (single phase)","softwareBuildID":"1.2.0","stackVersion":1,"type":"Router","zclVersion":1},"last_seen":"2024-09-04T03:21:57.906Z","linkquality":172,"state":"OFF","state_relay":"OFF"}'
info 2024-09-04 05:21:57z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/K: Relais-Bedienelement-Kombi rechts/relay', payload '{"brightness":254,"state":"OFF"}'
info 2024-09-04 05:21:58z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/G: Relais-Bedienelement-Kombi Bad aussen', payload '{"brightness_relay":254,"device":{"applicationVersion":1,"dateCode":"20161209","friendlyName":"G: Relais-Bedienelement-Kombi Bad aussen","hardwareVersion":0,"ieeeAddr":"0xd85def11a1003268","manufacturerID":4398,"manufacturerName":"Busch-Jaeger","model":"6735/6736/6737","networkAddress":6146,"powerSource":"Mains (single phase)","softwareBuildID":"1.2.0","stackVersion":1,"type":"Router","zclVersion":1},"last_seen":"2024-09-04T03:21:58.207Z","linkquality":255,"state":"OFF","state_relay":"OFF"}'
info 2024-09-04 05:21:58z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/G: Relais-Bedienelement-Kombi Bad aussen/relay', payload '{"brightness":254,"state":"OFF"}'
info 2024-09-04 05:21:58z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/FZ: Thermostat', payload '{"away_preset_temperature":"5.0","battery":100,"calibrate":null,"calibrated":true,"child_lock":true,"device":{"applicationVersion":25,"dateCode":"Feb 20 2023","friendlyName":"FZ: Thermostat","hardwareVersion":1,"ieeeAddr":"0x54ef4410007411b7","manufacturerID":4151,"manufacturerName":"LUMI","model":"SRTS-A01","networkAddress":25128,"powerSource":"Battery","softwareBuildID":"0.0.0_1030","type":"EndDevice","zclVersion":3},"device_temperature":26,"external_temperature_input":null,"internal_heating_setpoint":5,"last_seen":"2024-09-04T03:21:58.727Z","linkquality":188,"local_temperature":24.6,"occupied_heating_setpoint":5,"power_outage_count":44,"preset":"manual","schedule":false,"schedule_settings":"mon,tue,wed,thu,fri,sat,sun|3:00,21.0|5:30,19.0|14:00,21.0|17:30,19.0","sensor":"external","setup":false,"system_mode":"off","update":{"installed_version":2590,"latest_version":2590,"state":"idle"},"update_available":null,"valve_alarm":false,"valve_detection":true,"voltage":2800,"window_detection":true,"window_open":false}'
info 2024-09-04 05:21:58z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/G: Relais-Bedienelement-Kombi Bad aussen', payload '{"brightness_relay":254,"device":{"applicationVersion":1,"dateCode":"20161209","friendlyName":"G: Relais-Bedienelement-Kombi Bad aussen","hardwareVersion":0,"ieeeAddr":"0xd85def11a1003268","manufacturerID":4398,"manufacturerName":"Busch-Jaeger","model":"6735/6736/6737","networkAddress":6146,"powerSource":"Mains (single phase)","softwareBuildID":"1.2.0","stackVersion":1,"type":"Router","zclVersion":1},"last_seen":"2024-09-04T03:21:58.793Z","linkquality":255,"state":"OFF","state_relay":"OFF"}'
info 2024-09-04 05:21:58z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/G: Relais-Bedienelement-Kombi Bad aussen/relay', payload '{"brightness":254,"state":"OFF"}'
info 2024-09-04 05:21:58z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/K: Relais-Bedienelement-Kombi rechts', payload '{"brightness_relay":254,"device":{"applicationVersion":1,"dateCode":"20161209","friendlyName":"K: Relais-Bedienelement-Kombi rechts","hardwareVersion":0,"ieeeAddr":"0xd85def11a100469e","manufacturerID":4398,"manufacturerName":"Busch-Jaeger","model":"6735/6736/6737","networkAddress":65377,"powerSource":"Mains (single phase)","softwareBuildID":"1.2.0","stackVersion":1,"type":"Router","zclVersion":1},"last_seen":"2024-09-04T03:21:58.932Z","linkquality":172,"state":"OFF","state_relay":"OFF"}'
info 2024-09-04 05:21:58z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/K: Relais-Bedienelement-Kombi rechts/relay', payload '{"brightness":254,"state":"OFF"}'
info 2024-09-04 05:21:59z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/G: Relais-Bedienelement-Kombi Bad aussen', payload '{"brightness_relay":254,"device":{"applicationVersion":1,"dateCode":"20161209","friendlyName":"G: Relais-Bedienelement-Kombi Bad aussen","hardwareVersion":0,"ieeeAddr":"0xd85def11a1003268","manufacturerID":4398,"manufacturerName":"Busch-Jaeger","model":"6735/6736/6737","networkAddress":6146,"powerSource":"Mains (single phase)","softwareBuildID":"1.2.0","stackVersion":1,"type":"Router","zclVersion":1},"last_seen":"2024-09-04T03:21:59.018Z","linkquality":255,"state":"OFF","state_relay":"OFF"}'
info 2024-09-04 05:21:59z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/G: Relais-Bedienelement-Kombi Bad aussen/relay', payload '{"brightness":254,"state":"OFF"}'
info 2024-09-04 05:21:59z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/G: Relais-Bedienelement-Kombi Bad aussen', payload '{"brightness_relay":254,"device":{"applicationVersion":1,"dateCode":"20161209","friendlyName":"G: Relais-Bedienelement-Kombi Bad aussen","hardwareVersion":0,"ieeeAddr":"0xd85def11a1003268","manufacturerID":4398,"manufacturerName":"Busch-Jaeger","model":"6735/6736/6737","networkAddress":6146,"powerSource":"Mains (single phase)","softwareBuildID":"1.2.0","stackVersion":1,"type":"Router","zclVersion":1},"last_seen":"2024-09-04T03:21:59.319Z","linkquality":255,"state":"OFF","state_relay":"OFF"}'
info 2024-09-04 05:21:59z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/G: Relais-Bedienelement-Kombi Bad aussen/relay', payload '{"brightness":254,"state":"OFF"}'
info 2024-09-04 05:22:00z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/WZ: Relais-Bedienelement-Kombi Tür links', payload '{"brightness":254,"brightness_relay":254,"device":{"applicationVersion":1,"dateCode":"20161209","friendlyName":"WZ: Relais-Bedienelement-Kombi Tür links","hardwareVersion":0,"ieeeAddr":"0xd85def11a100458f","manufacturerID":4398,"manufacturerName":"Busch-Jaeger","model":"6735/6736/6737","networkAddress":52827,"powerSource":"Mains (single phase)","softwareBuildID":"1.2.0","stackVersion":1,"type":"Router","zclVersion":1},"last_seen":"2024-09-04T03:22:00.409Z","linkquality":168,"state":"OFF","state_relay":"OFF"}'
info 2024-09-04 05:22:00z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/WZ: Relais-Bedienelement-Kombi Tür links/relay', payload '{"brightness":254,"state":"OFF"}'
info 2024-09-04 05:22:02z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/WZ: Relais-Bedienelement-Kombi Tür links', payload '{"brightness":254,"brightness_relay":254,"device":{"applicationVersion":1,"dateCode":"20161209","friendlyName":"WZ: Relais-Bedienelement-Kombi Tür links","hardwareVersion":0,"ieeeAddr":"0xd85def11a100458f","manufacturerID":4398,"manufacturerName":"Busch-Jaeger","model":"6735/6736/6737","networkAddress":52827,"powerSource":"Mains (single phase)","softwareBuildID":"1.2.0","stackVersion":1,"type":"Router","zclVersion":1},"last_seen":"2024-09-04T03:22:02.080Z","linkquality":172,"state":"OFF","state_relay":"OFF"}'
info 2024-09-04 05:22:02z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/WZ: Relais-Bedienelement-Kombi Tür links/relay', payload '{"brightness":254,"state":"OFF"}'
info 2024-09-04 05:22:03z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/WZ: Relais-Bedienelement-Kombi Tür links', payload '{"brightness":254,"brightness_relay":254,"device":{"applicationVersion":1,"dateCode":"20161209","friendlyName":"WZ: Relais-Bedienelement-Kombi Tür links","hardwareVersion":0,"ieeeAddr":"0xd85def11a100458f","manufacturerID":4398,"manufacturerName":"Busch-Jaeger","model":"6735/6736/6737","networkAddress":52827,"powerSource":"Mains (single phase)","softwareBuildID":"1.2.0","stackVersion":1,"type":"Router","zclVersion":1},"last_seen":"2024-09-04T03:22:03.359Z","linkquality":172,"state":"OFF","state_relay":"OFF"}'
info 2024-09-04 05:22:03z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/WZ: Relais-Bedienelement-Kombi Tür links/relay', payload '{"brightness":254,"state":"OFF"}'
info 2024-09-04 05:22:04z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/WZ: Relais-Bedienelement-Kombi Tür links', payload '{"brightness":254,"brightness_relay":254,"device":{"applicationVersion":1,"dateCode":"20161209","friendlyName":"WZ: Relais-Bedienelement-Kombi Tür links","hardwareVersion":0,"ieeeAddr":"0xd85def11a100458f","manufacturerID":4398,"manufacturerName":"Busch-Jaeger","model":"6735/6736/6737","networkAddress":52827,"powerSource":"Mains (single phase)","softwareBuildID":"1.2.0","stackVersion":1,"type":"Router","zclVersion":1},"last_seen":"2024-09-04T03:22:04.893Z","linkquality":168,"state":"OFF","state_relay":"OFF"}'
info 2024-09-04 05:22:04z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/WZ: Relais-Bedienelement-Kombi Tür links/relay', payload '{"brightness":254,"state":"OFF"}'
info 2024-09-04 05:22:09z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/K: Relais-Bedienelement-Kombi Tür', payload '{"brightness_relay":254,"device":{"applicationVersion":1,"dateCode":"20161209","friendlyName":"K: Relais-Bedienelement-Kombi Tür","hardwareVersion":0,"ieeeAddr":"0xd85def11a100466a","manufacturerID":4398,"manufacturerName":"Busch-Jaeger","model":"6735/6736/6737","networkAddress":24447,"powerSource":"Mains (single phase)","softwareBuildID":"1.2.0","stackVersion":1,"type":"Router","zclVersion":1},"last_seen":"2024-09-04T03:22:09.173Z","linkquality":168,"state":"OFF","state_relay":"OFF"}'
info 2024-09-04 05:22:09z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/K: Relais-Bedienelement-Kombi Tür/relay', payload '{"brightness":254,"state":"OFF"}'
info 2024-09-04 05:22:09z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/K: Relais-Bedienelement-Kombi Tür', payload '{"brightness_relay":254,"device":{"applicationVersion":1,"dateCode":"20161209","friendlyName":"K: Relais-Bedienelement-Kombi Tür","hardwareVersion":0,"ieeeAddr":"0xd85def11a100466a","manufacturerID":4398,"manufacturerName":"Busch-Jaeger","model":"6735/6736/6737","networkAddress":24447,"powerSource":"Mains (single phase)","softwareBuildID":"1.2.0","stackVersion":1,"type":"Router","zclVersion":1},"last_seen":"2024-09-04T03:22:09.260Z","linkquality":168,"state":"OFF","state_relay":"OFF"}'
info 2024-09-04 05:22:09z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/K: Relais-Bedienelement-Kombi Tür/relay', payload '{"brightness":254,"state":"OFF"}'
info 2024-09-04 05:22:09z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/G: Relais-Bedienelement-Kombi Bad aussen', payload '{"brightness_relay":254,"device":{"applicationVersion":1,"dateCode":"20161209","friendlyName":"G: Relais-Bedienelement-Kombi Bad aussen","hardwareVersion":0,"ieeeAddr":"0xd85def11a1003268","manufacturerID":4398,"manufacturerName":"Busch-Jaeger","model":"6735/6736/6737","networkAddress":6146,"powerSource":"Mains (single phase)","softwareBuildID":"1.2.0","stackVersion":1,"type":"Router","zclVersion":1},"last_seen":"2024-09-04T03:22:09.390Z","linkquality":255,"state":"OFF","state_relay":"OFF"}'
info 2024-09-04 05:22:09z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/G: Relais-Bedienelement-Kombi Bad aussen/relay', payload '{"brightness":254,"state":"OFF"}'
info 2024-09-04 05:22:09z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/WZ: Deckenfluter', payload '{"brightness":254,"device":{"applicationVersion":64,"dateCode":"","friendlyName":"WZ: Deckenfluter","hardwareVersion":1,"ieeeAddr":"0xa4c13805b1e01b54","manufacturerID":4417,"manufacturerName":"_TZ3210_k1msuvg6","model":"TS110E_1gang_1","networkAddress":15404,"powerSource":"Mains (single phase)","stackVersion":0,"type":"Router","zclVersion":3},"last_seen":"2024-09-04T03:22:09.709Z","linkquality":148,"max_brightness":255,"min_brightness":4,"power_on_behavior":"off","state":"ON","switch_type":"toggle"}'
info 2024-09-04 05:22:09z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/G: Relais-Bedienelement-Kombi Bad aussen', payload '{"brightness_relay":254,"device":{"applicationVersion":1,"dateCode":"20161209","friendlyName":"G: Relais-Bedienelement-Kombi Bad aussen","hardwareVersion":0,"ieeeAddr":"0xd85def11a1003268","manufacturerID":4398,"manufacturerName":"Busch-Jaeger","model":"6735/6736/6737","networkAddress":6146,"powerSource":"Mains (single phase)","softwareBuildID":"1.2.0","stackVersion":1,"type":"Router","zclVersion":1},"last_seen":"2024-09-04T03:22:09.776Z","linkquality":255,"state":"OFF","state_relay":"OFF"}'
info 2024-09-04 05:22:09z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/G: Relais-Bedienelement-Kombi Bad aussen/relay', payload '{"brightness":254,"state":"OFF"}'
info 2024-09-04 05:22:10z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/G: Relais-Bedienelement-Kombi Bad aussen', payload '{"brightness_relay":254,"device":{"applicationVersion":1,"dateCode":"20161209","friendlyName":"G: Relais-Bedienelement-Kombi Bad aussen","hardwareVersion":0,"ieeeAddr":"0xd85def11a1003268","manufacturerID":4398,"manufacturerName":"Busch-Jaeger","model":"6735/6736/6737","networkAddress":6146,"powerSource":"Mains (single phase)","softwareBuildID":"1.2.0","stackVersion":1,"type":"Router","zclVersion":1},"last_seen":"2024-09-04T03:22:10.066Z","linkquality":255,"state":"OFF","state_relay":"OFF"}'
info 2024-09-04 05:22:10z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/G: Relais-Bedienelement-Kombi Bad aussen/relay', payload '{"brightness":254,"state":"OFF"}'
info 2024-09-04 05:22:10z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/G: Relais-Bedienelement-Kombi Bad aussen', payload '{"brightness_relay":254,"device":{"applicationVersion":1,"dateCode":"20161209","friendlyName":"G: Relais-Bedienelement-Kombi Bad aussen","hardwareVersion":0,"ieeeAddr":"0xd85def11a1003268","manufacturerID":4398,"manufacturerName":"Busch-Jaeger","model":"6735/6736/6737","networkAddress":6146,"powerSource":"Mains (single phase)","softwareBuildID":"1.2.0","stackVersion":1,"type":"Router","zclVersion":1},"last_seen":"2024-09-04T03:22:10.513Z","linkquality":255,"state":"OFF","state_relay":"OFF"}'
info 2024-09-04 05:22:10z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/G: Relais-Bedienelement-Kombi Bad aussen/relay', payload '{"brightness":254,"state":"OFF"}'
info 2024-09-04 05:22:10z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/FZ: Dimmer-Bedienelement-Kombi Tür', payload '{"brightness":254,"brightness_relay":54,"device":{"applicationVersion":1,"dateCode":"20161209","friendlyName":"FZ: Dimmer-Bedienelement-Kombi Tür","hardwareVersion":0,"ieeeAddr":"0xd85def11a1006a50","manufacturerID":4398,"manufacturerName":"Busch-Jaeger","model":"6735/6736/6737","networkAddress":37580,"powerSource":"Mains (single phase)","softwareBuildID":"1.2.0","stackVersion":1,"type":"Router","zclVersion":1},"last_seen":"2024-09-04T03:22:10.980Z","linkquality":192,"state":"ON","state_relay":"OFF"}'
info 2024-09-04 05:22:10z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/FZ: Dimmer-Bedienelement-Kombi Tür/relay', payload '{"brightness":54,"state":"OFF"}'
info 2024-09-04 05:22:11z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/MZ: Raumklima', payload '{"battery":100,"device":{"applicationVersion":5,"dateCode":"20211103","friendlyName":"MZ: Raumklima","hardwareVersion":1,"ieeeAddr":"0x00124b0029114286","manufacturerID":0,"manufacturerName":"eWeLink","model":"SNZB-02","networkAddress":51572,"powerSource":"Battery","type":"EndDevice","zclVersion":1},"humidity":66.54,"last_seen":"2024-09-04T03:22:11.262Z","linkquality":172,"temperature":25.56,"voltage":3000}'
info 2024-09-04 05:22:11z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/FZ: Dimmer-Bedienelement-Kombi Tür', payload '{"brightness":254,"brightness_relay":54,"device":{"applicationVersion":1,"dateCode":"20161209","friendlyName":"FZ: Dimmer-Bedienelement-Kombi Tür","hardwareVersion":0,"ieeeAddr":"0xd85def11a1006a50","manufacturerID":4398,"manufacturerName":"Busch-Jaeger","model":"6735/6736/6737","networkAddress":37580,"powerSource":"Mains (single phase)","softwareBuildID":"1.2.0","stackVersion":1,"type":"Router","zclVersion":1},"last_seen":"2024-09-04T03:22:11.508Z","linkquality":188,"state":"ON","state_relay":"OFF"}'
info 2024-09-04 05:22:11z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/FZ: Dimmer-Bedienelement-Kombi Tür/relay', payload '{"brightness":54,"state":"OFF"}'
error 2024-09-04 05:22:12z2m: Publish 'set' 'state' to 'WZ: Relais-Bedienelement-Kombi Tür links' failed: 'Error: ZCL command 0xd85def11a100458f/18 genLevelCtrl.moveToLevelWithOnOff({"level":254,"transtime":0}, {"timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"reservedBits":0,"writeUndiv":false}) failed ({"target":52827,"apsFrame":{"profileId":260,"clusterId":8,"sourceEndpoint":1,"destinationEndpoint":18,"options":4416,"groupId":0,"sequence":247},"zclSequence":141,"commandIdentifier":11} timed out after 10000ms)'
error 2024-09-04 05:22:12z2m: Publish 'set' 'state' to 'WZ: Dimmer-Bedienelement-Kombi Tür rechts' failed: 'Error: ZCL command 0xd85def11a10015b8/18 genLevelCtrl.moveToLevelWithOnOff({"level":254,"transtime":0}, {"timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"reservedBits":0,"writeUndiv":false}) failed (Delivery failed for {"profileId":260,"clusterId":8,"sourceEndpoint":1,"destinationEndpoint":18,"options":4160,"groupId":0,"sequence":249})'
info 2024-09-04 05:22:12z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/FZ: Dimmer-Bedienelement-Kombi Tür', payload '{"brightness":254,"brightness_relay":54,"device":{"applicationVersion":1,"dateCode":"20161209","friendlyName":"FZ: Dimmer-Bedienelement-Kombi Tür","hardwareVersion":0,"ieeeAddr":"0xd85def11a1006a50","manufacturerID":4398,"manufacturerName":"Busch-Jaeger","model":"6735/6736/6737","networkAddress":37580,"powerSource":"Mains (single phase)","softwareBuildID":"1.2.0","stackVersion":1,"type":"Router","zclVersion":1},"last_seen":"2024-09-04T03:22:12.579Z","linkquality":192,"state":"ON","state_relay":"OFF"}'
info 2024-09-04 05:22:12z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/FZ: Dimmer-Bedienelement-Kombi Tür/relay', payload '{"brightness":54,"state":"OFF"}'
info 2024-09-04 05:22:13z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/FZ: Dimmer-Bedienelement-Kombi Tür', payload '{"brightness":254,"brightness_relay":54,"device":{"applicationVersion":1,"dateCode":"20161209","friendlyName":"FZ: Dimmer-Bedienelement-Kombi Tür","hardwareVersion":0,"ieeeAddr":"0xd85def11a1006a50","manufacturerID":4398,"manufacturerName":"Busch-Jaeger","model":"6735/6736/6737","networkAddress":37580,"powerSource":"Mains (single phase)","softwareBuildID":"1.2.0","stackVersion":1,"type":"Router","zclVersion":1},"last_seen":"2024-09-04T03:22:13.241Z","linkquality":192,"state":"ON","state_relay":"OFF"}'
info 2024-09-04 05:22:13z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/FZ: Dimmer-Bedienelement-Kombi Tür/relay', payload '{"brightness":54,"state":"OFF"}'
info 2024-09-04 05:22:13z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/WZ: Thermostat 4', payload '{"away_preset_temperature":"5.0","battery":52,"calibrate":null,"calibrated":true,"child_lock":true,"device":{"applicationVersion":25,"dateCode":"Feb 20 2023","friendlyName":"WZ: Thermostat 4","hardwareVersion":1,"ieeeAddr":"0x54ef44100073cbb5","manufacturerID":4447,"manufacturerName":"LUMI","model":"SRTS-A01","networkAddress":32408,"powerSource":"Battery","softwareBuildID":"0.0.0_1030","type":"EndDevice","zclVersion":3},"device_temperature":25,"external_temperature_input":null,"internal_heating_setpoint":5,"last_seen":"2024-09-04T03:22:13.251Z","linkquality":172,"local_temperature":24,"occupied_heating_setpoint":5,"power_outage_count":0,"preset":"manual","schedule":false,"schedule_settings":"mon,sun|0:00,21.0|12:00,17.0|18:00,21.0|23:59,21.0","sensor":"external","setup":false,"system_mode":"off","update":{"installed_version":2590,"latest_version":2590,"state":"idle"},"update_available":null,"valve_alarm":true,"valve_detection":true,"voltage":3200,"window_detection":false,"window_open":false}'
info 2024-09-04 05:22:16z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/MZ: Dimmer-Bedienelement-Kombi Tür', payload '{"brightness_relay":254,"device":{"applicationVersion":1,"dateCode":"20161209","friendlyName":"MZ: Dimmer-Bedienelement-Kombi Tür","hardwareVersion":0,"ieeeAddr":"0xd85def11a1002561","manufacturerID":4398,"manufacturerName":"Busch-Jaeger","model":"6735/6736/6737","networkAddress":48322,"powerSource":"Mains (single phase)","softwareBuildID":"1.2.0","stackVersion":1,"type":"Router","zclVersion":1},"last_seen":"2024-09-04T03:22:16.630Z","linkquality":172,"state":"OFF","state_relay":"OFF"}'
info 2024-09-04 05:22:16z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/MZ: Dimmer-Bedienelement-Kombi Tür/relay', payload '{"brightness":254,"state":"OFF"}'
info 2024-09-04 05:22:16z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/MZ: Dimmer-Bedienelement-Kombi Tür', payload '{"brightness_relay":254,"device":{"applicationVersion":1,"dateCode":"20161209","friendlyName":"MZ: Dimmer-Bedienelement-Kombi Tür","hardwareVersion":0,"ieeeAddr":"0xd85def11a1002561","manufacturerID":4398,"manufacturerName":"Busch-Jaeger","model":"6735/6736/6737","networkAddress":48322,"powerSource":"Mains (single phase)","softwareBuildID":"1.2.0","stackVersion":1,"type":"Router","zclVersion":1},"last_seen":"2024-09-04T03:22:16.707Z","linkquality":168,"state":"OFF","state_relay":"OFF"}'
info 2024-09-04 05:22:16z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/MZ: Dimmer-Bedienelement-Kombi Tür/relay', payload '{"brightness":254,"state":"OFF"}'
info 2024-09-04 05:22:19z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/BK: Lounge', payload '{"brightness":110,"color_mode":"color_temp","color_temp":370,"device":{"applicationVersion":1,"dateCode":"20180404-42","friendlyName":"BK: Lounge","hardwareVersion":1,"ieeeAddr":"0x00158d000340fdbd","manufacturerID":4635,"manufacturerName":"MLI","model":"404006/404008/404004","networkAddress":50721,"powerSource":"Mains (single phase)","softwareBuildID":"2.0","stackVersion":1,"type":"Router","zclVersion":2},"last_seen":"2024-09-04T03:22:19.086Z","linkquality":144,"power_on_behavior":"off","state":"OFF"}'
info 2024-09-04 05:22:19z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/FZ: Raumklima', payload '{"battery":100,"device":{"applicationVersion":5,"dateCode":"20211103","friendlyName":"FZ: Raumklima","hardwareVersion":1,"ieeeAddr":"0x00124b0022d2cf25","manufacturerID":0,"manufacturerName":"eWeLink","model":"SNZB-02","networkAddress":46069,"powerSource":"Battery","type":"EndDevice","zclVersion":1},"humidity":68.89,"last_seen":"2024-09-04T03:22:19.208Z","linkquality":188,"temperature":25.51,"voltage":3000}'
info 2024-09-04 05:22:21z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/G: Relais-Bedienelement-Kombi Bad aussen', payload '{"brightness_relay":254,"device":{"applicationVersion":1,"dateCode":"20161209","friendlyName":"G: Relais-Bedienelement-Kombi Bad aussen","hardwareVersion":0,"ieeeAddr":"0xd85def11a1003268","manufacturerID":4398,"manufacturerName":"Busch-Jaeger","model":"6735/6736/6737","networkAddress":6146,"powerSource":"Mains (single phase)","softwareBuildID":"1.2.0","stackVersion":1,"type":"Router","zclVersion":1},"last_seen":"2024-09-04T03:22:21.146Z","linkquality":255,"state":"OFF","state_relay":"OFF"}'
info 2024-09-04 05:22:21z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/G: Relais-Bedienelement-Kombi Bad aussen/relay', payload '{"brightness":254,"state":"OFF"}'
info 2024-09-04 05:22:21z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/G: Relais-Bedienelement-Kombi Bad aussen', payload '{"brightness_relay":254,"device":{"applicationVersion":1,"dateCode":"20161209","friendlyName":"G: Relais-Bedienelement-Kombi Bad aussen","hardwareVersion":0,"ieeeAddr":"0xd85def11a1003268","manufacturerID":4398,"manufacturerName":"Busch-Jaeger","model":"6735/6736/6737","networkAddress":6146,"powerSource":"Mains (single phase)","softwareBuildID":"1.2.0","stackVersion":1,"type":"Router","zclVersion":1},"last_seen":"2024-09-04T03:22:21.819Z","linkquality":255,"state":"OFF","state_relay":"OFF"}'
info 2024-09-04 05:22:21z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/G: Relais-Bedienelement-Kombi Bad aussen/relay', payload '{"brightness":254,"state":"OFF"}'
info 2024-09-04 05:22:22z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/G: Relais-Bedienelement-Kombi Bad aussen', payload '{"brightness_relay":254,"device":{"applicationVersion":1,"dateCode":"20161209","friendlyName":"G: Relais-Bedienelement-Kombi Bad aussen","hardwareVersion":0,"ieeeAddr":"0xd85def11a1003268","manufacturerID":4398,"manufacturerName":"Busch-Jaeger","model":"6735/6736/6737","networkAddress":6146,"powerSource":"Mains (single phase)","softwareBuildID":"1.2.0","stackVersion":1,"type":"Router","zclVersion":1},"last_seen":"2024-09-04T03:22:22.745Z","linkquality":255,"state":"OFF","state_relay":"OFF"}'
info 2024-09-04 05:22:22z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/G: Relais-Bedienelement-Kombi Bad aussen/relay', payload '{"brightness":254,"state":"OFF"}'
info 2024-09-04 05:22:23z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/G: Relais-Bedienelement-Kombi Bad aussen', payload '{"brightness_relay":254,"device":{"applicationVersion":1,"dateCode":"20161209","friendlyName":"G: Relais-Bedienelement-Kombi Bad aussen","hardwareVersion":0,"ieeeAddr":"0xd85def11a1003268","manufacturerID":4398,"manufacturerName":"Busch-Jaeger","model":"6735/6736/6737","networkAddress":6146,"powerSource":"Mains (single phase)","softwareBuildID":"1.2.0","stackVersion":1,"type":"Router","zclVersion":1},"last_seen":"2024-09-04T03:22:23.590Z","linkquality":255,"state":"OFF","state_relay":"OFF"}'
info 2024-09-04 05:22:23z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/G: Relais-Bedienelement-Kombi Bad aussen/relay', payload '{"brightness":254,"state":"OFF"}'
info 2024-09-04 05:22:24z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/WC: Thermostat', payload '{"away_preset_temperature":"5.0","battery":61,"calibrate":null,"calibrated":true,"child_lock":false,"device":{"applicationVersion":25,"dateCode":"Feb 20 2023","friendlyName":"WC: Thermostat","hardwareVersion":1,"ieeeAddr":"0x54ef4410006ff481","manufacturerID":4151,"manufacturerName":"LUMI","model":"SRTS-A01","networkAddress":26507,"powerSource":"Battery","softwareBuildID":"0.0.0_1030","type":"EndDevice","zclVersion":3},"device_temperature":28,"external_temperature_input":null,"internal_heating_setpoint":5,"last_seen":"2024-09-04T03:22:24.657Z","linkquality":168,"local_temperature":25.3,"occupied_heating_setpoint":5,"power_outage_count":1,"preset":"manual","schedule":false,"schedule_settings":"mon,sun|0:00,21.0|12:00,17.0|18:00,21.0|23:59,21.0","sensor":"internal","setup":false,"system_mode":"off","update":{"installed_version":2590,"latest_version":2590,"state":"idle"},"update_available":null,"valve_alarm":false,"valve_detection":true,"voltage":2900,"window_detection":false,"window_open":false}'
info 2024-09-04 05:22:29z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/K: Relais-Bedienelement-Kombi Tür', payload '{"brightness_relay":254,"device":{"applicationVersion":1,"dateCode":"20161209","friendlyName":"K: Relais-Bedienelement-Kombi Tür","hardwareVersion":0,"ieeeAddr":"0xd85def11a100466a","manufacturerID":4398,"manufacturerName":"Busch-Jaeger","model":"6735/6736/6737","networkAddress":24447,"powerSource":"Mains (single phase)","softwareBuildID":"1.2.0","stackVersion":1,"type":"Router","zclVersion":1},"last_seen":"2024-09-04T03:22:29.334Z","linkquality":172,"state":"OFF","state_relay":"OFF"}'
info 2024-09-04 05:22:29z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/K: Relais-Bedienelement-Kombi Tür/relay', payload '{"brightness":254,"state":"OFF"}'
info 2024-09-04 05:22:29z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/K: Relais-Bedienelement-Kombi Tür', payload '{"brightness_relay":254,"device":{"applicationVersion":1,"dateCode":"20161209","friendlyName":"K: Relais-Bedienelement-Kombi Tür","hardwareVersion":0,"ieeeAddr":"0xd85def11a100466a","manufacturerID":4398,"manufacturerName":"Busch-Jaeger","model":"6735/6736/6737","networkAddress":24447,"powerSource":"Mains (single phase)","softwareBuildID":"1.2.0","stackVersion":1,"type":"Router","zclVersion":1},"last_seen":"2024-09-04T03:22:29.413Z","linkquality":172,"state":"OFF","state_relay":"OFF"}'
info 2024-09-04 05:22:29z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/K: Relais-Bedienelement-Kombi Tür/relay', payload '{"brightness":254,"state":"OFF"}'

```

**Comment by Ra72xx** [comment]
I also noticed that the "last seen" value in the ui often gets (much) hight than the configured polling intervall. That does not seem correct to me, as polling means querying and "seeing" the device.

**Comment by Ra72xx** [comment]
Update: I THOUGHT I had the problem solved: I re-learned all Busch-Jäger Zigbee devices and afterwards everything went smooth. But after a restart, everything is worse than before, log is flooded with error messages:

`Sep 20 19:56:13 Obelix zigbee2mqtt[48268]:     at processImmediate (node:internal/timers:476:21))
Sep 20 19:56:13 Obelix zigbee2mqtt[48268]: [2024-09-20 19:56:13] error:         z2m: Failed to configure 'FZ: Dimmer-Bedienelement-Kombi Tür', attempt 2 (Error: Bind 0xd85def11a1006a50/18 genLevelCtrl from '0x6c5cb1fffeed6c26/1' failed ([ZDO] Failed response for '44729' cluster '32801' Status 'TABLE_FULL'.)
Sep 20 19:56:13 Obelix zigbee2mqtt[48268]:     at EmberOneWaitress.resolveZDO (/app/node_modules/zigbee-herdsman/src/adapter/ember/adapter/oneWaitress.ts:139:35)
Sep 20 19:56:13 Obelix zigbee2mqtt[48268]:     at EmberAdapter.onZDOResponse (/app/node_modules/zigbee-herdsman/src/adapter/ember/adapter/emberAdapter.ts:556:30)
Sep 20 19:56:13 Obelix zigbee2mqtt[48268]:     at Ezsp.emit (node:events:517:28)
Sep 20 19:56:13 Obelix zigbee2mqtt[48268]:     at Ezsp.ezspIncomingMessageHandler (/app/node_modules/zigbee-herdsman/src/adapter/ember/ezsp/ezsp.ts:5249:18)
Sep 20 19:56:13 Obelix zigbee2mqtt[48268]:     at Ezsp.callbackDispatch (/app/node_modules/zigbee-herdsman/src/adapter/ember/ezsp/ezsp.ts:782:26)
Sep 20 19:56:13 Obelix zigbee2mqtt[48268]:     at Ezsp.onAshFrame (/app/node_modules/zigbee-herdsman/src/adapter/ember/ezsp/ezsp.ts:398:22)
Sep 20 19:56:13 Obelix zigbee2mqtt[48268]:     at UartAsh.emit (node:events:517:28)
Sep 20 19:56:13 Obelix zigbee2mqtt[48268]:     at Immediate.<anonymous> (/app/node_modules/zigbee-herdsman/src/adapter/ember/uart/ash.ts:1169:45)
Sep 20 19:56:13 Obelix zigbee2mqtt[48268]:     at processImmediate (node:internal/timers:476:21))
Sep 20 19:56:24 Obelix zigbee2mqtt[48268]: [2024-09-20 19:56:24] error:         z2m: Failed to unbind cluster 'genOnOff' from 'WZ: Relais-Bedienelement-Kombi Tür links' to 'Coordinator' (Error: Unbind 0xd85def11a100458f/18 genOnOff from '0x6c5cb1fffeed6c26/1' failed (Delivery failed for {"profileId":0,"clusterId":34,"sourceEndpoint":0,"destinationEndpoint":0,"options":4160,"groupId":0,"sequence":227}))
Sep 20 19:56:24 Obelix zigbee2mqtt[48268]: [2024-09-20 19:56:24] error:         z2m: Failed to unbind
Sep 20 19:56:33 Obelix zigbee2mqtt[48268]: [2024-09-20 19:56:33] error:         z2m: Failed to configure 'FZ: Dimmer-Bedienelement-Kombi Tür', attempt 3 (Error: Bind 0xd85def11a1006a50/18 genLevelCtrl from '0x6c5cb1fffeed6c26/1' failed ([ZDO] Failed response for '44729' cluster '32801' Status 'TABLE_FULL'.)
Sep 20 19:56:33 Obelix zigbee2mqtt[48268]:     at EmberOneWaitress.resolveZDO (/app/node_modules/zigbee-herdsman/src/adapter/ember/adapter/oneWaitress.ts:139:35)
Sep 20 19:56:33 Obelix zigbee2mqtt[48268]:     at EmberAdapter.onZDOResponse (/app/node_modules/zigbee-herdsman/src/adapter/ember/adapter/emberAdapter.ts:556:30)
Sep 20 19:56:33 Obelix zigbee2mqtt[48268]:     at Ezsp.emit (node:events:517:28)
Sep 20 19:56:33 Obelix zigbee2mqtt[48268]:     at Ezsp.ezspIncomingMessageHandler (/app/node_modules/zigbee-herdsman/src/adapter/ember/ezsp/ezsp.ts:5249:18)
Sep 20 19:56:33 Obelix zigbee2mqtt[48268]:     at Ezsp.callbackDispatch (/app/node_modules/zigbee-herdsman/src/adapter/ember/ezsp/ezsp.ts:782:26)
Sep 20 19:56:33 Obelix zigbee2mqtt[48268]:     at Ezsp.onAshFrame (/app/node_modules/zigbee-herdsman/src/adapter/ember/ezsp/ezsp.ts:398:22)
Sep 20 19:56:33 Obelix zigbee2mqtt[48268]:     at UartAsh.emit (node:events:517:28)
Sep 20 19:56:33 Obelix zigbee2mqtt[48268]:     at Immediate.<anonymous> (/app/node_modules/zigbee-herdsman/src/adapter/ember/uart/ash.ts:1169:45)
Sep 20 19:56:33 Obelix zigbee2mqtt[48268]:     at processImmediate (node:internal/timers:476:21))
Sep 20 19:57:13 Obelix zigbee2mqtt[48268]: [2024-09-20 19:57:13] error:         z2m: Failed to configure 'B: Thermostat', attempt 2 (Error: ZCL command 0x54ef4410007043f7/1 manuSpecificLumi.read([1034], {"timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"reservedBits":0,"manufacturerCode":4447,"writeUndiv":false}) failed (Delivery failed for {"profileId":260,"clusterId":64704,"sourceEndpoint":1,"destinationEndpoint":1,"options":4160,"groupId":0,"sequence":15})
Sep 20 19:57:13 Obelix zigbee2mqtt[48268]:     at EmberOneWaitress.deliveryFailedFor (/app/node_modules/zigbee-herdsman/src/adapter/ember/adapter/oneWaitress.ts:101:31)
Sep 20 19:57:13 Obelix zigbee2mqtt[48268]:     at EmberAdapter.onMessageSent (/app/node_modules/zigbee-herdsman/src/adapter/ember/adapter/emberAdapter.ts:480:42)
Sep 20 19:57:13 Obelix zigbee2mqtt[48268]:     at Ezsp.emit (node:events:517:28)
Sep 20 19:57:13 Obelix zigbee2mqtt[48268]:     at Ezsp.ezspMessageSentHandler (/app/node_modules/zigbee-herdsman/src/adapter/ember/ezsp/ezsp.ts:4982:14)
Sep 20 19:57:13 Obelix zigbee2mqtt[48268]:     at Ezsp.callbackDispatch (/app/node_modules/zigbee-herdsman/src/adapter/ember/ezsp/ezsp.ts:738:26)
Sep 20 19:57:13 Obelix zigbee2mqtt[48268]:     at Ezsp.onAshFrame (/app/node_modules/zigbee-herdsman/src/adapter/ember/ezsp/ezsp.ts:398:22)
Sep 20 19:57:13 Obelix zigbee2mqtt[48268]:     at UartAsh.emit (node:events:517:28)
Sep 20 19:57:13 Obelix zigbee2mqtt[48268]:     at Immediate.<anonymous> (/app/node_modules/zigbee-herdsman/src/adapter/ember/uart/ash.ts:1169:45)
Sep 20 19:57:13 Obelix zigbee2mqtt[48268]:     at processImmediate (node:internal/timers:476:21))
Sep 20 19:57:30 Obelix zigbee2mqtt[48268]: [2024-09-20 19:57:30] error:         z2m: Failed to configure 'WZ: Relais-Bedienelement-Kombi Tür links', attempt 1 (Error: Bind 0xd85def11a100458f/18 genLevelCtrl from '0x6c5cb1fffeed6c26/1' failed ([ZDO] Failed response for '36627' cluster '32801' Status 'TABLE_FULL'.)
Sep 20 19:57:30 Obelix zigbee2mqtt[48268]:     at EmberOneWaitress.resolveZDO (/app/node_modules/zigbee-herdsman/src/adapter/ember/adapter/oneWaitress.ts:139:35)
Sep 20 19:57:30 Obelix zigbee2mqtt[48268]:     at EmberAdapter.onZDOResponse (/app/node_modules/zigbee-herdsman/src/adapter/ember/adapter/emberAdapter.ts:556:30)
Sep 20 19:57:30 Obelix zigbee2mqtt[48268]:     at Ezsp.emit (node:events:517:28)
Sep 20 19:57:30 Obelix zigbee2mqtt[48268]:     at Ezsp.ezspIncomingMessageHandler (/app/node_modules/zigbee-herdsman/src/adapter/ember/ezsp/ezsp.ts:5249:18)
Sep 20 19:57:30 Obelix zigbee2mqtt[48268]:     at Ezsp.callbackDispatch (/app/node_modules/zigbee-herdsman/src/adapter/ember/ezsp/ezsp.ts:782:26)
Sep 20 19:57:30 Obelix zigbee2mqtt[48268]:     at Ezsp.onAshFrame (/app/node_modules/zigbee-herdsman/src/adapter/ember/ezsp/ezsp.ts:398:22)
Sep 20 19:57:30 Obelix zigbee2mqtt[48268]:     at UartAsh.emit (node:events:517:28)
Sep 20 19:57:30 Obelix zigbee2mqtt[48268]:     at Immediate.<anonymous> (/app/node_modules/zigbee-herdsman/src/adapter/ember/uart/ash.ts:1169:45)
Sep 20 19:57:30 Obelix zigbee2mqtt[48268]:     at processImmediate (node:internal/timers:476:21))
Sep 20 19:57:33 Obelix zigbee2mqtt[48268]: [2024-09-20 19:57:33] error:         z2m: Failed to configure 'WZ: Relais-Bedienelement-Kombi Tür links', attempt 2 (Error: Bind 0xd85def11a100458f/18 genLevelCtrl from '0x6c5cb1fffeed6c26/1' failed ([ZDO] Failed response for '36627' cluster '32801' Status 'TABLE_FULL'.)
Sep 20 19:57:33 Obelix zigbee2mqtt[48268]:     at EmberOneWaitress.resolveZDO (/app/node_modules/zigbee-herdsman/src/adapter/ember/adapter/oneWaitress.ts:139:35)
Sep 20 19:57:33 Obelix zigbee2mqtt[48268]:     at EmberAdapter.onZDOResponse (/app/node_modules/zigbee-herdsman/src/adapter/ember/adapter/emberAdapter.ts:556:30)
Sep 20 19:57:33 Obelix zigbee2mqtt[48268]:     at Ezsp.emit (node:events:517:28)
Sep 20 19:57:33 Obelix zigbee2mqtt[48268]:     at Ezsp.ezspIncomingMessageHandler (/app/node_modules/zigbee-herdsman/src/adapter/ember/ezsp/ezsp.ts:5249:18)
Sep 20 19:57:33 Obelix zigbee2mqtt[48268]:     at Ezsp.callbackDispatch (/app/node_modules/zigbee-herdsman/src/adapter/ember/ezsp/ezsp.ts:782:26)
Sep 20 19:57:33 Obelix zigbee2mqtt[48268]:     at Ezsp.onAshFrame (/app/node_modules/zigbee-herdsman/src/adapter/ember/ezsp/ezsp.ts:398:22)
Sep 20 19:57:33 Obelix zigbee2mqtt[48268]:     at UartAsh.emit (node:events:517:28)
Sep 20 19:57:33 Obelix zigbee2mqtt[48268]:     at Immediate.<anonymous> (/app/node_modules/zigbee-herdsman/src/adapter/ember/uart/ash.ts:1169:45)
Sep 20 19:57:33 Obelix zigbee2mqtt[48268]:     at processImmediate (node:internal/timers:476:21))
Sep 20 19:57:50 Obelix zigbee2mqtt[48268]: [2024-09-20 19:57:50] error:         z2m: Failed to configure 'WZ: Relais-Bedienelement-Kombi Tür links', attempt 3 (Error: Bind 0xd85def11a100458f/18 genLevelCtrl from '0x6c5cb1fffeed6c26/1' failed ([ZDO] Failed response for '36627' cluster '32801' Status 'TABLE_FULL'.)
Sep 20 19:57:50 Obelix zigbee2mqtt[48268]:     at EmberOneWaitress.resolveZDO (/app/node_modules/zigbee-herdsman/src/adapter/ember/adapter/oneWaitress.ts:139:35)
Sep 20 19:57:50 Obelix zigbee2mqtt[48268]:     at EmberAdapter.onZDOResponse (/app/node_modules/zigbee-herdsman/src/adapter/ember/adapter/emberAdapter.ts:556:30)
Sep 20 19:57:50 Obelix zigbee2mqtt[48268]:     at Ezsp.emit (node:events:517:28)
Sep 20 19:57:50 Obelix zigbee2mqtt[48268]:     at Ezsp.ezspIncomingMessageHandler (/app/node_modules/zigbee-herdsman/src/adapter/ember/ezsp/ezsp.ts:5249:18)
Sep 20 19:57:50 Obelix zigbee2mqtt[48268]:     at Ezsp.callbackDispatch (/app/node_modules/zigbee-herdsman/src/adapter/ember/ezsp/ezsp.ts:782:26)
Sep 20 19:57:50 Obelix zigbee2mqtt[48268]:     at Ezsp.onAshFrame (/app/node_modules/zigbee-herdsman/src/adapter/ember/ezsp/ezsp.ts:398:22)
Sep 20 19:57:50 Obelix zigbee2mqtt[48268]:     at UartAsh.emit (node:events:517:28)
Sep 20 19:57:50 Obelix zigbee2mqtt[48268]:     at Immediate.<anonymous> (/app/node_modules/zigbee-herdsman/src/adapter/ember/uart/ash.ts:1169:45)
Sep 20 19:57:50 Obelix zigbee2mqtt[48268]:     at processImmediate (node:internal/timers:476:21))
Sep 20 19:58:12 Obelix zigbee2mqtt[48268]: [2024-09-20 19:58:12] error:         z2m: Failed to configure 'B: Thermostat', attempt 3 (Error: ZCL command 0x54ef4410007043f7/1 manuSpecificLumi.read([1034], {"timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"reservedBits":0,"manufacturerCode":4447,"writeUndiv":false}) failed (Delivery failed for {"profileId":260,"clusterId":64704,"sourceEndpoint":1,"destinationEndpoint":1,"options":4160,"groupId":0,"sequence":72})
Sep 20 19:58:12 Obelix zigbee2mqtt[48268]:     at EmberOneWaitress.deliveryFailedFor (/app/node_modules/zigbee-herdsman/src/adapter/ember/adapter/oneWaitress.ts:101:31)
Sep 20 19:58:12 Obelix zigbee2mqtt[48268]:     at EmberAdapter.onMessageSent (/app/node_modules/zigbee-herdsman/src/adapter/ember/adapter/emberAdapter.ts:480:42)
Sep 20 19:58:12 Obelix zigbee2mqtt[48268]:     at Ezsp.emit (node:events:517:28)
Sep 20 19:58:12 Obelix zigbee2mqtt[48268]:     at Ezsp.ezspMessageSentHandler (/app/node_modules/zigbee-herdsman/src/adapter/ember/ezsp/ezsp.ts:4982:14)
Sep 20 19:58:12 Obelix zigbee2mqtt[48268]:     at Ezsp.callbackDispatch (/app/node_modules/zigbee-herdsman/src/adapter/ember/ezsp/ezsp.ts:738:26)
Sep 20 19:58:12 Obelix zigbee2mqtt[48268]:     at Ezsp.onAshFrame (/app/node_modules/zigbee-herdsman/src/adapter/ember/ezsp/ezsp.ts:398:22)
Sep 20 19:58:12 Obelix zigbee2mqtt[48268]:     at UartAsh.emit (node:events:517:28)
Sep 20 19:58:12 Obelix zigbee2mqtt[48268]:     at Immediate.<anonymous> (/app/node_modules/zigbee-herdsman/src/adapter/ember/uart/ash.ts:1169:45)
Sep 20 19:58:12 Obelix zigbee2mqtt[48268]:     at processImmediate (node:internal/timers:476:21))
`

**Comment by Ra72xx** [comment]
For me this issue is not stale. The BJ devices are really slow to react. If it gets to bad, re-pairing them makes them work instantly again, but after every restart of Z2M chances are that one of the devices starts having these extremely long delays again.

**Comment by klada** [comment]
I can reproduce the `TABLE_FULL` errors during initial device configuration now. I don't understand why they never showed up before, maybe because my network was smaller and it just kept growing. I think the `TABLE_FULL` errors will show up again for you if you restart zigbee2mqtt as it tries to reconfigure devices which failed to configure before. This might also explain why you are experiencing these delays after a z2m restart.

`TABLE_FULL` is an error from the BJ device. It indicates that a device has exhausted it's memory for bindings (and possible other things) and I am pretty sure that this is due to the size of the ZigBee network. In the device specification from Busch-Jaeger they mention that their devices support networks with (only) up to 20 devices. That's not a lot.

We can try to work around these errors by skipping some bindings. Could you please provide the following details for the device :

- Which kind of control panel is installed on "FZ: Dimmer-Bedienelement-Kombi Tür". Does it have 1, 2 or 4 rows?
- Did you place the "FZ: Dimmer-Bedienelement-Kombi Tür" or any other BJ devices in a ZigBee group? If yes, in how many different ZigBee groups did you put these devices?

I will try to create an external converter for testing purposes but I need to know these details first. Hopefully we can get rid of the error message and we'll see if this could also affect stability in your environment.

You have mentioned that you were using a SkyConnect and migrated to a different coordinator. Would it be possible for you to fire up a fresh test Z2M instance on your laptop/PC with the SkyConnect and pair only a single Busch-Jaeger device? I currently don't have access to a second coordinator which makes these experiments kind of hard for me 😄 I do expect that this would work and that the size of the network is the actual issue with these devices. Let's hope we can do something about `TABLE_FULL` at least.

To be clear: I think the issues you are having with the BJ devices is not related to any of the recent Z2M/converter changes. It's very likely related to network size, possible in combination with the coordinator type/firmware.

**Comment by Ra72xx** [comment]
Thanks for picking this subject up! Are we currently talking about the same problem in different threads ;-) ? I will close the other one... 

The error messages at startup seem to have no further negative effect.

>     * Which kind of control panel is installed on "FZ: Dimmer-Bedienelement-Kombi Tür". Does it have 1, 2 or 4 rows?

Most of those devices in my network including this one are in the 4 row variant. The problems/delays happen for the 2 row variant all the same, however.
 
>     * Did you place the "FZ: Dimmer-Bedienelement-Kombi Tür" or any other BJ devices in a ZigBee group? If yes, in how many different ZigBee groups did you put these devices?

No, I don't use any Zigbee groups. Rows 2 to 4 are programmed using Home Assistant automations (with row 1 directly operating the relais/dimmer).

> You have mentioned that you were using a SkyConnect and migrated to a different coordinator. Would it be possible for you to fire up a fresh test Z2M instance on your laptop/PC with the SkyConnect and pair only a single Busch-Jaeger device?

I have sold the old coordinator already, sorry.

Something which I also noticed: After re-pairing, the bindings often also have to be recreated at first (as described in the Z2M device page for the BJ stuff), i.e. source endpoints 10 to 13 have to be connected to the coordinator. In some cases, after some time also a binding from source endpoint 18 is suddenly created (can be deleted again, seems not to matter).

**Comment by klada** [comment]
> EDIT @klada: The https://github.com/Koenkk/zigbee2mqtt/issues/23347#issue-2410057118 is open, while this one is formally closed already. Which one should we keep open?

Just keep the other one open as well as there was a message from another user with (IMHO) a different issue. I personally cannot repoen this issue as I am not part of the Z2M project.

> Most of those devices in my network including this one are in the 4 row variant.

OK, good.

> The problems/delays happen for the 2 row variant all the same, however.

That not so good news. The potentially fixable `TABLE_FULL` errors should be related to the 4 row variants exclusively. But let's still give it a try. I'll get back to you in a few days.

> No, I don't use any Zigbee groups. 

OK, good. That saves some memory in the devices :)

> Something which I also noticed: After re-pairing, the bindings often also have to be recreated at first (as described in the Z2M device page for the BJ stuff), i.e. source endpoints 10 to 13 have to be connected to the coordinator. In some cases, after some time also a binding from source endpoint 18 is suddenly created (can be deleted again, seems not to matter).

Yes, this is caused by the `TABLE_FULL` errors. And BTW, you should not bind EP10 to the coordinators for dimmers/relays. You just need that endpoint for power supplies.

The binding to EP18 is the one I'd like to get rid of. As you said it seems to be not required and if it helps to free up some memory in these devices we should remove it. Again, just give me a few days for this.

**Comment by klada** [comment]
OK, I did some small modifications to the BJ converter which you can test.

Could you please follow these steps:

1. Create an `external_converters` directory in your zigbee2mqtt data dir
2. Put the following file in it:  https://gist.github.com/klada/f124c64cf49fb5571f9448044f1ec794
3. Restart zigbee2mqtt and make sure it is generally working. If the custom converter is active you should be able to see that the device description of the BJ devices has changed to "Zigbee Light Link power supply/relay/dimmer/wall-switch **test**"
4. Remove one of your BJ devices from zigbee2mqtt. The device will reset itself to factory defaults.
5. Stop zigbee2mqtt
6. Remove your ZigBee USB dongle and replug it (this makes sure the firmware of your ZigBee coordinator also reboots)
7. Remove power from the one BJ device you have removed (through circuit breaker) and repower it after a few seconds. This makes sure the BJ device reboots completely
8. Reset the BJ device again through the key combination (press both buttons of top row, wait until LEDs start blinking, press both top-row buttons again)
9. start zigbee2mqtt, wait for ~30 seconds for the ZigBee network to settle and enable joining mode

It would be interesting to see if the device is still showing this behavior or if it does improve things.

I have tested the converter myself and in my environment it fixes the `TABLE_FULL` errors.

**Comment by Ra72xx** [comment]
I'm a bit reluctant to experiment with my - currently working - setup and will do it once I have time to re-create the network if it works out wrong ;-). All that resetting with those BJ devices is a very cumbersome job...

A few questions:
1. The custom converter will indeed be applied automatically for the BJ devices after step 3? What are all the other steps good for?
2. Do I have to do the reset procedure for each of my BJ devices, or is it (for this test) enough to try one of them (and the others will at least keep on working as before)?
3. Do I have to do this procedure again once your patch is hopefully good enough to be included in Z2M?

**Comment by klada** [comment]
>     1. The custom converter will indeed be applied automatically for the BJ devices after step 3? 

Yes

> What are all the other steps good for?

The converter does not change runtime-behavior. It changes the way these BJ devices are configured after pairing them. So to notice anything you'd need to reset a BJ device in order to use the new configuration parameters.

>     2. Do I have to do the reset procedure for each of my BJ devices, or is it (for this test) enough to try one of them (and the others will at least keep on working as before)?

For testing it is enough to test this with a single BJ device. That's also the way I am testing this in my (production) environment.

>     3. Do I have to do this procedure again once your patch is hopefully good enough to be included in Z2M?

No. If a configured device is working as intended you don't have to go through the reset procedure again.

**Comment by Ra72xx** [comment]
I tried your fix with 2 of my 4-row devices and indeed

- the bindings for those two were at once correct without manual interference
- the TABLE_FULL error did not occur - however also not for the other, unchanged, devices?!

**Comment by klada** [comment]
>  * the TABLE_FULL error did not occur

This is good, thanks for testing. This means that the fix itself is working. Do you experience anything else? Are all the button presses reported properly to Z2M? Did the reaction time improve or is it still the same? My guess is that it's still the same but at least these `TABLE_FULL` messages are gone now... 

>    however also not for the other, unchanged, devices?!

This is good, I did not expect that. But on the other hand it makes sense. I think that they went through reconfiguration as well (because it was not completed successfully earlier) and now they are probably also marked as configured. I'd say that this is at least something 😄

**Comment by Ra72xx** [comment]
I did not yet notice any other effects, neither positive nor negative. As the delay was always "variable", I cannot really say if it has gotten better.

**Comment by Ra72xx** [comment]
Thanks for committing the fix. To be be a bit pendantic, it fixes the "table full" error messages at startup, but it basically does not fix this issue here: The delays are still there. So I'd keep open this bug?

**Comment by klada** [comment]
OK, that's what I was afraid of. Thanks for testing anyways. At least the configuration issue is solved.

I am afraid we cannot do much on the zigbee2mqtt-side. The reason for the slowdowns is probably somewhere within your ZigBee network. As I said the BJ devices have never been designed to be used in big networks, that could be a reason. Or there is a device in your network which is triggering this issue.

I personally don't believe that the configurable polling interval is the culprit either. You could check that by going to an old Z2M version, but my guess is that this is too much work for likely zero benefit. In the end it's probably easier to get another Coordinator and create another small ZigBee network for just the BJ devices. Or ditch those devices entirely, although I know that there is no good alternative for them out there on the market. Sorry.

BTW: I remember having issues with the BJ devices with Deconz a long time ago. They were also behaving badly, like dropping out of the network after a power outage. That was the reason why I switched to Z2M and they have worked much better ever since.

**Comment by Ra72xx** [comment]
Unfortunately, there are no real alternatives if you want the Zigbee switches integrated (design-wise) with the rest of the electric installation. But you somehow get used to the delay after some time.
